[E 250225 11:51:32 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '689958'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001DF22676990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 11:51:40 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '689958'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000001F91F186990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 11:51:53 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '689958'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000017427D86990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 11:51:58 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '689958'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000023D95AF6990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 11:52:33 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '335525'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002CE8FD36990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 11:53:01 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '633742'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000024B4F176990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 12:05:18 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '587183'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002598ADE6990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 12:06:02 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '247506'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000187FA936990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
[E 250225 12:14:48 smartConnect:221] Error occurred while making a POST request to https://apiconnect.angelone.in/rest/auth/angelbroking/user/v1/loginByPassword. Headers: {'Content-type': 'application/json', 'X-ClientLocalIP': '127.0.0.1', 'X-ClientPublicIP': '106.193.147.98', 'X-MACAddress': 'f8:a2:d6:02:c2:bd', 'Accept': 'application/json', 'X-PrivateKey': 'uQBIgMGv', 'X-UserType': 'USER', 'X-SourceID': 'WEB'}, Request: {'clientcode': 'S692948', 'password': '5472', 'totp': '278301'}, Response: HTTPSConnectionPool(host='apiconnect.angelone.in', port=443): Max retries exceeded with url: /rest/auth/angelbroking/user/v1/loginByPassword (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000219D85F6990>: Failed to resolve 'apiconnect.angelone.in' ([Errno 11001] getaddrinfo failed)"))
